routing/pathfind.go

```diff
@@ -956,7 +1017,7 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 			}
 
 			edge := edgeUnifier.getEdge(
```

Comment:
I don't think that parallel policy logic is all that essential, because node operators should know that ambiguous policies may lead to failures. But nevertheless you are right. I am now passing in `nextOutFee` to `edgeUnifier`.